src/wp-admin/user-edit.php

```diff
@@ -733,6 +733,7 @@
 
 	</table>
 
+<?php if ( wp_is_application_passwords_available_for_user( $user_id ) || ! wp_is_application_passwords_supported() ) : // phpcs:disable Generic.WhiteSpace.ScopeIndent ?>
```

Comment:
Indentation in this file is off. The `phpcs:disable` is being added in order to focus this PR on only the needed changes that resolve the reported issue. Formatting can be resolved in a follow-up PR/ticket.